src/util.js

```diff
@@ -82,7 +82,7 @@ export function monitor(name, args, callback, on_close, system) {
                 if (e.name === "ConnectionClosed")
                     on_close(system);
                 else
-                    console.log(e);
+                    return Promise.reject(e);
```

Comment:
Instead of converting one exception (IOW, rejected promise) into another, `throw e;` would be a bit more straightforward and also keep the original stack trace.

Reply:
I saw this commit https://github.com/cockpit-project/cockpit-podman/commit/2818fa6aae171701edd6ac684809ee184c71f758 and thought I would do the same. But if `throw` is better, I have no problem converting it.

Reply:
Ah, I was not aware of that commit. @KKoukiou may know the details then. It just looks strange at first sight.

Reply:
You can use throw if you want. I already checked and we handle with catch any possible exceptions from these two functions so there is no danger of OOps here.
Also the is not code after this line, so stopping the control flow, which is what `throw` will do will not have any impact here.